build.ps1

```diff
@@ -50,7 +50,7 @@ cmake `
 	-DLLVM_INCLUDE_TOOLS=ON `
 	-DLLVM_INCLUDE_UTILS=OFF `
 	-DLLVM_OPTIMIZED_TABLEGEN=ON `
-	-DLLVM_TARGETS_TO_BUILD="X86;AArch64" `
+	-DLLVM_STATIC_LINK_CXX_STDLIB=ON `
```

Comment:
[Quoting the documentation](https://www.llvm.org/docs/CMake.html):

> LLVM_STATIC_LINK_CXX_STDLIB:BOOL
>    Statically link to the C++ standard library if possible. This uses the flag “-static-libstdc++”, but a Clang host compiler will statically link to libc++ if used in conjunction with the LLVM_ENABLE_LIBCXX flag. Defaults to OFF.

I don't know exactly the implications of this on Windows.

I think we should also take into account comments from @nlewycky https://github.com/wasmerio/wasmer/pull/2151.

Reply:
From my testing, this option does not make a difference actually, it works either way w/ or w/o it.